HouseRules_Essentials/README.md

```diff
@@ -1,35 +1,244 @@
-# RulesAPI Rules
+# HouseRules Essentials
 
-A collection of predefined rules and rulesets for RulesAPI.
+A collection of predefined rules and rulesets for HouseRules_Core
 
-See the [RulesAPI readme](../RulesAPI_Core/README.md) for more information about
+See the [HouseRules_Core readme](../HouseRules_Core/README.md) for more information about
 RulesAPI.
 
+The game is is played using a board, a number of __Pieces__ and a number of __Abilities__
+
+## Rulesets
+
+### Built-in Rulesets
+
+- **SampleRuleset**: A [sample ruleset](https://github.com/orendain/DemeoMods/blob/045aec568fdddb95b63a1ed34abcb64065e4ca99/Rules/RulesMod.cs#L27-L28)
+  for the purposes of documenting how to create a ruleset (and for testing during development).
+
+### JSON Rulesets
+
+Rulesets may also be defined in JSON within the MelonLoader config file. A sample [MelonLoader config with JSON ruleset](sample-json-ruleset.txt) is provided to help you get started. 
+
```

Comment:
No issue with this text, I just have a related question:

Would it be helpful if we allowed JSON rulesets to exist as their own file?  E.g.: `ruleset = "SampleRuleset"` would imply that the ruleset can be found in a file called `SampleRuleset.json` in some directory (e.g., `/UserData/HouseRules/`)?

Just thinking aloud - figured I'd share before I forgot.

Reply:
Oh hell yes... most definitely.. That would be brilliant.... We can put a few examples/built-ins in there... It would be very helpful for new people trying to work out how to create their own rules too.

Reply:
Implemented here: https://github.com/orendain/DemeoMods/pull/134